Documentation/Decode-Enums.md

```diff
@@ -0,0 +1,69 @@
+# Decoding Enums
+
+Decoding Structs and Classes is a similar process, but Enums can mix things up.
```

Comment:
I think it is suppose to be `is` because of `Decoding` being the verb. I think `Decoding ... is a similar process.` where the object is singular as opposed to `Structs and Classes are similar things` where the object is plural ... however, my english isn't world renown so I will default to @gabebw s thoughts.